.eslintrc

```diff
@@ -3,5 +3,8 @@
     "parserOptions": {
         "sourceType": "module",
         "requireConfigFile": false
+    },
+    "rules": {
+      "no-sync": "off"
```

Comment:
@fraxken The new `getTokenSync` method triggers eslint `no-sync` error. I'm not sure if we should disable this warning in `@nodesecure/eslint-config` as the rule is fine but it's like our package is incompatible with our linter..  Wdyt ?

Reply:
Yes I'm ok with this (since we are in CLI that's totally ok).